soda/core/soda/scan.py

```diff
@@ -479,19 +479,28 @@ def execute(self) -> int:
                 # Validate check attributes as well.
                 if check.check_cfg.source_configurations:
                     if self._configuration.soda_cloud:
-                        attributes = check.check_cfg.source_configurations.get("attributes", {})
-                        attribute_handler = AttributeHandler(self._logs)
-                        self._logs.info("Validating check attributes.")
-                        # TODO: get attributes schema from Cloud Configuration when it is supported by cloud.
-                        # attributes_schema = self._configuration.soda_cloud.get_check_attributes_schema()
-                        attributes_schema = {}
-
-                        attributes, invalid_attributes = attribute_handler.validate(attributes, attributes_schema)
-
-                        if invalid_attributes:
-                            self._logs.debug(f"Skipping invalid attributes '{invalid_attributes.keys()}'.")
-
-                        check.attributes = attributes
+                        check_attributes = check.check_cfg.source_configurations.get("attributes", {})
+
+                        if check_attributes:
```

Comment:
This is a bit clumsy:
1. I commented out the actual validation code so that current users can still use the attributes syntax without any effect, they just get set to the check.attributes property as-is for now
2. this whole section will need to be moved to earlier stage so that scan can be halted, this will be done later when cloud is ready